editor/scene_tree_dock.cpp

```diff
@@ -1886,7 +1894,9 @@ void SceneTreeDock::_tree_rmb(const Vector2 &p_menu_pos) {
 		menu->add_icon_shortcut(get_icon("ScriptCreate", "EditorIcons"), ED_GET_SHORTCUT("scene_tree/attach_script"), TOOL_ATTACH_SCRIPT);
 		menu->add_icon_shortcut(get_icon("ScriptRemove", "EditorIcons"), ED_GET_SHORTCUT("scene_tree/clear_script"), TOOL_CLEAR_SCRIPT);
 		menu->add_separator();
+		menu->add_icon_shortcut(get_icon("Rename", "EditorIcons"), ED_GET_SHORTCUT("scene_tree/rename"), TOOL_RENAME);
 	}
+	menu->add_icon_shortcut(get_icon("Rename", "EditorIcons"), ED_GET_SHORTCUT("scene_tree/batch_rename"), TOOL_BATCH_RENAME);
```

Comment:
It should probably be moved to an `else`, otherwise we'll get two "Rename" options for single selections.

Reply:
Or is Batch Rename useful for single selections too?

Reply:
Hey I commented down below, but to elaborate (sorry, not used to Github code review)

On batch rename, I'm not entirely sure there's any point to use it on a single node since I'm not the author of that functionality. I merely kept it outside of an else as that's how it was when I looked at the code.

Reply:
I've had a look and Batch Rename is not useful for a single node, so you can put it in an `else` branch IMO.